src/common/security/admiral/context.go

```diff
@@ -64,6 +64,11 @@ func (s *SecurityContext) IsSysAdmin() bool {
 	return s.ctx.IsSysAdmin()
 }
 
+// PolicyCheck ...
```

Comment:
I don't think we need this, we can modify the `Can` func for robot security context and make it return true for a special action.